<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userManager</class>
 <widget class="QMainWindow" name="userManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="glUserManager">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bNewUser">
        <property name="text">
         <string>New User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditUser">
        <property name="text">
         <string>Edit User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bDelUser">
        <property name="text">
         <string>Del User</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="twTree">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
